src/bin/bench-streamer.rs

```diff
@@ -54,28 +62,83 @@ fn sink(
     })
 }
 
+macro_rules! socketaddr {
```

Comment:
Instead of duplicating this code, can you add `#[macro_use]` above `extern crate solana;`?